packages/block-editor/src/components/alignment-control/stories/index.story.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { AlignmentControl } from '../';
+
+/**
+ * The `AlignmentControl` component renders a dropdown menu that displays alignment options for the selected block.
+ *
+ * This component is mostly used for blocks that display text, such as Heading, Paragraph, Post Author, Post Comments, Verse, Quote, Post Title, etc... And the available alignment options are `left`, `center` or `right` alignment.
+ *
+ * If you want to use the alignment control in a toolbar, you should use the `AlignmentToolbar` component instead.
+ */
+const meta = {
+	title: 'BlockEditor/AlignmentControl',
+	component: AlignmentControl,
+	argTypes: {
+		value: {
+			control: {
+				type: 'select',
+				options: [ 'undefined', 'left', 'center', 'right' ],
```

Comment:
Why is the `select` type being used here? I'm not familiar with these parameters, but it doesn't seem to be reflected in Storybook at all 🤔

Reply:
Oh, Sorry. I forgot to erase what I wrote to try it out....